jgrapht-core/src/main/java/org/jgrapht/alg/cycle/ChordalGraphMinimalVertexSeparatorFinder.java

```diff
@@ -74,12 +75,15 @@
     /**
      * A mapping of minimal separators to their multiplicities
      */
-    private Map<Set<V>, Integer> minimalSeparators;
+    private Map<Set<V>, Integer> minimalSeparatorsWithMultiplicities;
+    /**
+     * Set of minimal separators
+     */
+    private Set<Set<V>> minimalSeparators;
```

Comment:
there's no need to maintain a separate set here. You could simply return minimalSeparatorsWithMultiplicities.keySet() right?